src/_help/enhance-documentation-content/markdown-support.md

```diff
@@ -5,7 +5,11 @@ title: Markdown support
 - TOC
 {:toc}
 
-Bump supports common Markdown syntax, language color syntax highlighting, and information call-outs. Markdown can be included inside your contract file or as an external reference using dedicated Markdown files. Here's several options that may help.
+The generation of your API documentation is based on your API document, which must adhere to one of the standards we support (OpenAPI/Swagger and AsyncAPI.).
```

Comment:
```suggestion
The generation of your API documentation is based on your API definition, which must adhere to one of the standards we support (OpenAPI/Swagger and AsyncAPI).
```

Reply:
Works with the 2. `document` is OK here, but `definition` works better imo 👍